FirebaseAdmin/FirebaseAdmin.Tests/Auth/FirebaseUserManagerTest.cs

```diff
@@ -165,5 +222,64 @@ public async Task UpdateUserHttpError()
             await Assert.ThrowsAsync<FirebaseException>(
                 async () => await userManager.UpdateUserAsync(new UserRecord("user1") { CustomClaims = customClaims }));
         }
+
+        [Fact]
+        public async Task DeleteUser()
+        {
+            var handler = new MockMessageHandler()
+            {
+                Response = new Dictionary<string, string>()
+                {
+                    { "kind", "identitytoolkit#DeleteAccountResponse" },
+                },
+            };
+            var factory = new MockHttpClientFactory(handler);
+            var userManager = new FirebaseUserManager(
+                new FirebaseUserManagerArgs
+                {
+                    Credential = MockCredential,
+                    ProjectId = MockProjectId,
+                    ClientFactory = factory,
+                });
+            await userManager.DeleteUser("user1");
+        }
+
+        [Fact]
+        public async Task DeleteUserNotFound()
+        {
+            var handler = new MockMessageHandler()
+            {
+                StatusCode = HttpStatusCode.NotFound,
+            };
+            var factory = new MockHttpClientFactory(handler);
+            var userManager = new FirebaseUserManager(
+                new FirebaseUserManagerArgs
+                {
+                    Credential = MockCredential,
+                    ProjectId = MockProjectId,
+                    ClientFactory = factory,
+                });
+            await Assert.ThrowsAsync<FirebaseException>(
+               async () => await userManager.DeleteUser("user1"));
+        }
+
+        [Fact]
+        public async Task DeleteUserInvalidToken()
```

Comment:
Here we can remove one of these test cases. They test the same thing.

Reply:
I thought so. Will do